### PR TITLE
Timestamp documentation 

### PR DESCRIPTION
**Pull request by JustinXHale**

### CODE REVIEW

**Review by JustinXHale — COMMENTED** [review]
removed line breaks

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+id: Timestamp
+section: components
+---
+
+## Elements
+
+<img src="./img/timestamp-with-tooltip-underline.png" alt="Timestamp with tooltip underline"/>
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:8]: Need to add width to these images bc they're super blown up right now... should be the width of the artboard they're on in sketch!

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+<img src="./img/timestamp-tooltip-popover.png" alt="Timestamp with tooltip popover"/>
+
+1. **Day:** Spelled out or abbreviated format. [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics)
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:11]: Add a full stop at the end of each of these bullets

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### Best practices
+#### For timestamp:
+- Depending on the context, show either date or time, or both date and time.
+- Locale should be appropriately set so that users see dates and times in a familiar format.
+- Timestamp at minimum should include date, time, and timezone (shorthand format).
+- Every element does not need to be used when implementing a timestamp.
+#### For tooltip:
+- Not required but recommended to provide more precise time for relative times. Tooltips should show timestamps in UTC time.
+- Recommend displaying time in 24-hour format
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:30]: Add full stop

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Displays time in UTC (Coordinated Universal Time).
+- Should be formatted the same as the timestamp. 
+
+See UX [numerics guidelines](https://www.patternfly.org/v4/ux-writing/numerics) for more information about relative time best practices.
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:34]: Delete the word "UX" here

**Review by mmenestr — CHANGES_REQUESTED** [review]
Looks good overall!! Just left a few comments. Also need to add a width to all your images!
Also going to get content to review these, so up to you if you want to hold off on addressing my comments until they take a look

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Variations
+### 12-hour format
+In the 12-hour format use AM to refer to a time of day before noon, and PM to speak of a time between noon and midnight.
```
Comment (wolfeallison) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:38]: I'd add a comma after "format"

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+<img src="./img/timestamp-am-pm-format.png" alt="Timestamp AM/PM format"/>
+
+### 24-hour format
+In the 24-hour clock format, each day runs from midnight to midnight and is divided into 24 hours, starting at 1 a.m.
```
Comment (wolfeallison) [inline packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:43]: Doesn't it start at midnight, with 00:00? I think it is confusing saying it runs from midnight to midnight and then say it starts at 1. What about " each day runs from midnight to midnight and is divided into 24 hours, beginning with 00:00 AM at midnight."

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md (hunk 1 above)
Comment (JustinXHale) [reply packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:8]: no problem. We should update the size in the design documentation bc it only states a minimum size. 
image

File: packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md (hunk 6 above)
Comment (JustinXHale) [reply packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md:43]: sounds good @wolfeallison. Midnight is a 0 & 24 hour which can be confusing as well, which is why I chose to use this specific definition.  I'm fine with the definition you've written out, will update.

**Review by JustinXHale — COMMENTED** [review]
Update image widths to 868.
Add full stop to end of bullets.
Updated 24 hour explanation.

**Review by JustinXHale — COMMENTED** [review]
updated

**Pull request closed without merging**